onnxscript/rewriter/onnxruntime/transformers/biassplitgelu.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+import logging
+
+import onnx
+
+import onnxscript
+from onnxscript.rewriter import function_rule
+
+logger = logging.getLogger(__name__)
+
+
+class GegluRewriteRule(function_rule.FunctionRewriteRule):
```

Comment:
Is this a typo? Is it meant to be `Gelu...`?

Reply:
Same below

Reply:
It's an activation function in diffuser: 
https://github.com/huggingface/diffusers/blob/e23c27e90524c3c3e93525d03f18408b84372012/src/diffusers/models/activations.py#L77